localization-proofreading-english/StringTableEnExaminations.xml

```diff
@@ -101,76 +101,76 @@
+            <GameDBLocalizedString>      <LocID>EXM_X_RAY_HEAD_DESCRIPTION</LocID>                              <Text>X-ray imaging of the bones of the skull. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_X_RAY_LOWER_LIMB</LocID>                                    <Text>X-ray lower limb</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_X_RAY_LOWER_LIMB_DESCRIPTION</LocID>                        <Text>X-ray imaging of the bones and shadows of some tissues of lower limb. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_X_RAY_TORSO</LocID>                                         <Text>X-ray torso</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_X_RAY_TORSO_DESCRIPTION</LocID>                             <Text>X-ray imaging of the bones and shadows of some tissues of torso. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_X_RAY_LOWER_LIMB_DESCRIPTION</LocID>                        <Text>X-ray imaging of the bones of the lower limb. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_X_RAY_TORSO</LocID>                                         <Text>X-ray chest</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_X_RAY_TORSO_DESCRIPTION</LocID>                             <Text>X-ray imaging of the bones and shadows of the thorax. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
             <GameDBLocalizedString>      <LocID>EXM_X_RAY_UPPER_LIMB</LocID>                                    <Text>X-ray upper limb</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_X_RAY_UPPER_LIMB_DESCRIPTION</LocID>                        <Text>X-ray imaging of the bones and shadows of some tissues of upper limb. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
+            <GameDBLocalizedString>      <LocID>EXM_X_RAY_UPPER_LIMB_DESCRIPTION</LocID>                        <Text>X-ray imaging of the bones the arm. \n\nRequired room: X-ray room</Text>  </GameDBLocalizedString>
 
             <GameDBLocalizedString>      <LocID>EXM_DIFFERENTIAL_DIAGNOSIS</LocID>                              <Text>Differential diagnosis</Text>  </GameDBLocalizedString>
-            <GameDBLocalizedString>      <LocID>EXM_DIFFERENTIAL_DIAGNOSIS_DESCRIPTION</LocID>                  <Text>Special type of examination that could be used only by the doctor with Advance diagnosis specialization. It reduces total number of possible final diagnosis.</Text>  </GameDBLocalizedString>
```

Comment:
Linje 165: Omformuleret, men skal måske lige tjekkes om den stadig passer gameplay-wise. Jeg har ikke ændret på kravene til lægen, som jeg antager kan være problemet i spillet? - Den anden formulering var bare kaotisk og unøjagtig.

Reply:
Jeg synes dette er bedre, vær dog opmærksom på det overflødige "of" i din rettede tekst.